src/pages/FooterPages/ContributorPage/index.css

```diff
@@ -0,0 +1,104 @@
+.contributor-container {
+  position: relative;
+  margin-top: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.contributor-sub-container {
+  position: relative;
+  border-radius: 150px;
+  margin-bottom: 50px;
+}
+
+.contributor-section {
+  position: relative;
+  width: 70vw;
+  min-height: 100vh;
+  border-radius: 60px;
+  padding-bottom: 40px;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  align-items: center;
+}
+
+.grad5 {
```

Comment:
aren't these supposed to be in design.css ?

Reply:
Yes, but I use this only on the contributor page so I write this code in this CSS file.

Reply:
if we add them in design.css we can use them anywhere?

Reply:
yes